torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -74,6 +74,10 @@ inline Tensor celu(Tensor& input, const CELUOptions& options = {}) {
   }
 }
 
+inline Tensor sigmoid(const Tensor& input) {
+  return torch::sigmoid(input);
+}
```

Comment:
We can probably remove `F::sigmoid`, because the Python version mentions that `nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.`